go-controller/pkg/ovn/base_network_controller_kubevirt.go

```diff
@@ -0,0 +1,63 @@
+package ovn
+
+import (
+	"fmt"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kubevirt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+)
+
+func (bnc *BaseNetworkController) ensureNetworkInfoForVM(origPod *corev1.Pod) error {
+	if !kubevirt.AllowPodBridgeNetworkLiveMigration(origPod.Annotations) {
+		return nil
+	}
+	vmNetworkInfo, err := kubevirt.FindNetworkInfo(bnc.watchFactory, origPod)
```

Comment:
nit: would it make sense to use a struct for the kubevirt thing ? It would allow you to not have to pass the watch factory to every func call. 

Would also allow you you mock it when providing tests.

Reply:
Maybe the right thing to do is to add those functions to `BaseNetworkController` like the other `base_network_controller` do, but this is related to kubevirt.